### PR TITLE
Add OCI related jobs to JOBOWNERS to notify BARX

### PR DESCRIPTION
**Pull request by Pythyu**
### What does this PR do?

Modify `JOBOWNERS` to add `agent-build-and-releases` as owner of OCI related jobs

### CODE REVIEW

File: .gitlab/JOBOWNERS [hunk 1]
```diff
@@ -41,10 +41,10 @@ agent_suse*                          @DataDog/agent-build-and-releases
 iot_agent_suse*                      @DataDog/agent-build-and-releases
 dogstatsd_suse*                      @DataDog/agent-build-and-releases
 agent_dmg*                           @DataDog/agent-build-and-releases
+*oci*                                @DataDog/agent-build-and-releases
 windows_msi*                         @DataDog/windows-agent
 
 # Packaging
-agent_oci                            @DataDog/agent-build-and-releases
 updater_deb-*                        @DataDog/agent-build-and-releases
```
Comment (chouquette) [inline .gitlab/JOBOWNERS:48]: those would need to be renamed to `installer_*` (although it's not strictly related to your PR)

**Review by chouquette — APPROVED** [review]

**Review by KevinFairise2 — APPROVED** [review]

**Review by amenasria — APPROVED** [review]
LGTM !

**Pull request merged**